config/src/main/java/org/jboss/pnc/bacon/config/Config.java

```diff
@@ -65,10 +65,8 @@ public static void configure(String configLocation) throws IOException {
     public static void initialize() throws IOException {
         File configFile = new File(configLocation);
         if (!configFile.exists()) {
-            log.error(
-                    "Config file: '{}' does not exist. Please create a config file and either name it 'config.yaml' and put it in the working directory or specify it with -Dconfig",
-                    configLocation);
-            throw new IOException("Config file " + configLocation + " does not exist!");
```

Comment:
We were logging missing of config file twice, once here and once when catching exception